resources/user-guides/Importing_Blockchain.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: static_page
+title: Importing the Monero Blockchain from an outside source 
+---
+
+# Importing the Blockchain to Monero GUI wallet (Windows)
+
+### Step 1
+
+Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw, you can skip this step if you are importing the Blockchain from another source.
+
+### Step 2
+
+Find the Path of your Monero wallet (the folder where you extracted your wallet) for example mine is.
+
+`D:\monero-gui-0.10.3.1-beta2`
+
+Yours might be different depending on where you decided to download your wallet and what version of the Monero wallet you have.
+
+### Step 3
+
+Find the path of your downloaded Blockchain for example mine was. 
+
+`C:\Users\KeeJef\Downloads\blockchain.raw`
+
+Yours might be different depending on where you downloaded the Blockchain to.
+
+### Step 4
+
+Open a Command Prompt window. You can do this by pressing the Windows key + R, and then typing in the popup box `CMD`
+
+### Step 5
+
+Now you need to navigate using the CMD window to the path of your Monero wallet. You can do this by typing. 
+
+`cd THE PATH OF YOUR MONERO WALLET HERE` 

```

Comment:
> THE PATH OF YOUR MONERO WALLET HERE

Consider changing this to something that makes it obvious you're talking about a file path and use it throughout the document. i.e. C:\PATH\OF\YOUR\MONERO\WALLET.